package.json

```
@@ -5,7 +5,7 @@
   "description": "Foreman isn't really a node module, these are just dependencies needed to build the webpack bundle. 'dependencies' are the asset libraries in use and 'devDependencies' are used for the build process.",
   "private": true,
   "engines": {
-    "node": "<16.0.0"
+    "node": ">12.0.0 <16.0.0"
```

I think it's safe to raise this to 14 by now
```suggestion
    "node": ">14.0.0 <16.0.0"
```